src/EditorFeatures/Test/EditorConfig/LegacyEditorConfigDocumentOptionsProviderTests.cs

```diff
@@ -59,7 +59,7 @@ public void OrderingOfEditorConfigMaintained()
             var optionKey = new OptionKey(option);
 
             // Fetch the underlying option order with a "option" that returns the keys
-            provider.GetOptionsForDocumentAsync(document, CancellationToken.None).Result.TryGetDocumentOption(optionKey, workspace.Options, out object actualKeysInOrderObject);
+            provider.GetOptionsForDocumentAsync(document, CancellationToken.None).Result.TryGetDocumentOption(optionKey, out object actualKeysInOrderObject);
```

Comment:
I understand that this commit is an old commit you've brought forward, but it may make sense to just delete this test entirely, at least in a follow up commit -- hopefully with this PR we're no longer sensitive to ordering, or at least if we still are it's a case where we wouldn't expect the underlying services to maintain the ordering.

(In other words, it's odd that if I look at this PR and ignore the one it builds on, that this PR is updating a test which we're obsoleting.)